app/main/views/search.py

```diff
@@ -204,6 +195,21 @@ def supplier_search():
 
     pages = get_page_list(SUPPLIER_RESULTS_PER_PAGE, num_results, page)
 
+    SELLER_TYPE_KEYS = [
+        'start_up',
+        'sme',
+        'nfp_social_enterprise',
+        'product',
+        'indigenous'
+    ]
+
+    seller_type_filters = {st: st in selected_seller_types for st in SELLER_TYPE_KEYS}
```

Comment:
ignore my earlier comment, this is already getting used to populate the option list, so is required.